api/proto/docker_registry.proto

```diff
@@ -0,0 +1,94 @@
+syntax = "proto3";
+package proto;
+
+import "permissions.proto";
+
+// Configuration for the Docker Registry provider.
+message DockerRegistry {
+    // Whether the provider is enabled.
+    bool enabled = 1;
+
+    // The list of configured accounts.
+    repeated DockerRegistryAccount accounts = 2;
+
+    // The name of the primary account.
+    string primaryAccount = 3;
+}
+
+// A credential able to authenticate against a set of Docker repositories.
+message DockerRegistryAccount {
+    // The name of the account.
+    string name = 1;
+
+    // (Required) The registry address from which to pull and deploy images
+    // (e.g., `index.docker.io`).
```

Comment:
I have this configured as `https://gcr.io` (ie, including the protocol)...does it also work without the protocol, or should we include that in the example here?

Reply:
Great callout, [Halyard adds `https://` if neither that nor `http://` is included in the address](https://github.com/spinnaker/halyard/blob/master/halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/providers/dockerRegistry/DockerRegistryAccount.java#L51), but for kleat let's make users include the protocol (updated the example here and added to the list of validations/defaults/breaking changes).